helpers/arguments.js

```diff
@@ -0,0 +1,23 @@
+
+module.exports.MODULE = '--module';
+
+/**
+ * Check to make sure we have an element name
+ */
+module.exports.ArgumentsHaveElementName = () => {
```

Comment:
You're using .NET/Microsoft conventions for method names lol, this should be `argumentsHaveElementName`.